.ado/templates/apple-xcode-build-static-libs.yml

```diff
@@ -14,62 +14,62 @@ steps:
-      xcode_workspacePath: 'apps/ios/src/FluentTester.xcworkspace'
+      xcode_sdk: 'macosx'
+      xcode_workspacePath: 'apps/macos/src/FluentTester.xcworkspace'
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Debug'
       xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig'
 
-  # iPhone Simulator Release
+  # macOS Release
   - template: apple-xcode-build.yml
     parameters:
-      xcode_sdk: 'iphonesimulator'
-      xcode_workspacePath: 'apps/ios/src/FluentTester.xcworkspace'
+      xcode_sdk: 'macosx'
+      xcode_workspacePath: 'apps/macos/src/FluentTester.xcworkspace'
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Release'
       xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig'
 
-  # iPhone Device Debug
+  # iPhone Simulator Debug
   - template: apple-xcode-build.yml
     parameters:
-      xcode_sdk: 'iphoneos'
+      xcode_sdk: 'iphonesimulator'
       xcode_workspacePath: 'apps/ios/src/FluentTester.xcworkspace'
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Debug'
-      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides_ios_device.xcconfig'
+      xcode_extraArgs: ''
 
-  # iPhone Device Release
+  # iPhone Simulator Release
   - template: apple-xcode-build.yml
     parameters:
-      xcode_sdk: 'iphoneos'
+      xcode_sdk: 'iphonesimulator'
       xcode_workspacePath: 'apps/ios/src/FluentTester.xcworkspace'
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Release'
-      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides_ios_device.xcconfig'
+      xcode_extraArgs: ''
 
-  # macOS Debug
+  # iPhone Device Debug
   - template: apple-xcode-build.yml
     parameters:
-      xcode_sdk: 'macosx'
-      xcode_workspacePath: 'apps/macos/src/FluentTester.xcworkspace'
+      xcode_sdk: 'iphoneos'
+      xcode_workspacePath: 'apps/ios/src/FluentTester.xcworkspace'
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Debug'
-      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig' ONLY_ACTIVE_ARCH=NO
+      xcode_extraArgs: ''
 
-  # macOS Release
+  # iPhone Device Release
   - template: apple-xcode-build.yml
     parameters:
-      xcode_sdk: 'macosx'
-      xcode_workspacePath: 'apps/macos/src/FluentTester.xcworkspace'
+      xcode_sdk: 'iphoneos'
+      xcode_workspacePath: 'apps/ios/src/FluentTester.xcworkspace'
       xcode_actions: 'build'
       xcode_scheme: 'FluentTester'
       xcode_configuration: 'Release'
-      xcode_extraArgs: '-xcconfig $(Build.Repository.LocalPath)/.ado/xcconfig/publish_overrides.xcconfig' ONLY_ACTIVE_ARCH=NO
+      xcode_extraArgs: ''
```

Comment:
I only moved two steps from the bottom to the top (to build macOS first). I guess because the steps are all so similar, the diff makes it look a lot worse than it is.

Reply:
Is it necessary to revert the order back to building iOS first?

Reply:
It is not necessary, the order does not matter.